safedelete/queryset.py

```diff
@@ -16,7 +38,7 @@ def __init__(self, model=None, query=None, using=None, hints=None):
         super(SafeDeleteQueryset, self).__init__(model=model, query=query, using=using, hints=hints)
         self.query = query or SafeDeleteQuery(self.model)
 
-    def delete(self, force_policy=None):
+    def delete(self, force_policy=None, is_pre_signalize=False):
```

Comment:
I'd rename `is_pre_signalize` to something more meaningfull, such as `send_pre_signal`.